grails-app/i18n/messages.properties

```diff
@@ -2814,30 +2814,47 @@ react.stockListManagement.alert.noManagerAssociated.label=There is no manager as
 react.stockListManagement.alert.noManagerSelected.label=Please add a manager as a recipient and resend.
 
 # Dashboard titles
-react.dashboard.numberData.inventoryByLotAndBin.label=Inventory Details by Lot and Bin
-react.dashboard.numberData.inProgressShipments.label=Your in Progress Shipments
-react.dashboard.numberData.inProgressPutaways.label=Your in Progress Putaways
-react.dashboard.numberData.receivingBin.label=Products in Receiving Bin
-react.dashboard.numberData.itemsInventoried.label=Items Inventoried this Month
-react.dashboard.numberData.defaultBin.label=Products in Default Bin
-react.dashboard.numberData.productWithNegativeInventory.label=Products with Negative Inventory
-react.dashboard.numberData.expiredProductsInStock.label=Expired products in Stock
+react.dashboard.inventoryByLotAndBin.title.label=Inventory Details by Lot and Bin
+react.dashboard.inventoryByLotAndBin.info.label=Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque gravida non erat sed gravida. Vestibulum auctor condimentum elit eget tristique. Nullam lorem lectus, rhoncus sit amet convallis id, varius at orci
```

Comment:
`Lorem ipsum...`? Should not this be some other type of text in these labels?

Reply:
This was to be able to test the definitions since I didn't know the expected text. Should I remove them all for now?

Reply:
Is it possible to get the proper labels from Kelsey before merging this one?